src/App.tsx

```diff
@@ -167,7 +178,8 @@ class Board extends React.Component<BoardProps, BoardState> {
 
     public render() {
         return (
-            <div className="game" onKeyPress={this.handlePress} tabIndex={0}>
+            <div className="game" /*onKeyDown={this.handleKeyDown} tabIndex={0}*/>
+                <KeyDownListener onKeyDown={this.handleKeyDown}/>
```

Comment:
This needs to be bound properly

Reply:
I thought using arrow functions properly binds it.